ecpy_hqc_legacy/tasks/tasks/instr/views/dc_views.enaml

```diff
@@ -1,30 +1,29 @@
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
-# Copyright 2015-2016 by EcpyHqcLegacy Authors, see AUTHORS for more details.
+# Copyright 2016-2017 by EcpyHqcLegacy Authors, see AUTHORS for more details.
 #
 # Distributed under the terms of the BSD license.
 #
 # The full license is in the file LICENCE, distributed with this software.
 # -----------------------------------------------------------------------------
-"""Views for the DC sources related tasks.
+"""Views for the microwave sources related tasks.
 
 """
 from __future__ import (division, unicode_literals, print_function,
                         absolute_import)
 
-from textwrap import fill
-
-from enaml.core.api import Conditional, Include
+from enaml.core.api import Conditional
 from enaml.widgets.api import (GroupBox, Label, Field, ObjectCombo, CheckBox)
 from enaml.stdlib.fields import FloatField, IntField
 from enaml.layout.api import factory
 
-from ecpy.tasks.api import EVALUATER_TOOLTIP
+from textwrap import fill
```

Comment:
Please standard library import should be first (cf guidelines)

Reply:
I copied this from rf_views do you mean the fill import needs to come last?

Reply:
The fill import should come first because textwrap is part of the stdlib.